dotCMS/src/main/java/com/dotcms/util/HttpRequestDataUtil.java

```diff
@@ -32,6 +39,8 @@
  */
 public class HttpRequestDataUtil {
 
+	private static String serverPort = null;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'serverPort'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)